azure-pipelines.yml

```
@@ -0,0 +1,72 @@
+parameters:
+  - name: sendgridEnvironments
+    type: object
+    default:
+    - nonprod
+    - prod
+
+trigger:
+  - master
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+stages:
+  - ${{ each environment in parameters.sendgridEnvironments }}:
+    - stage: ${{ environment }}
+
+      variables:
+        isMain: $[eq(variables['Build.SourceBranch'], 'refs/heads/master')]
+        location: uksouth    
+        terraformVersion: 0.14.8
+        serviceConnection: dts-management-${{ environment }}-intsvc
+        resourceGroup: mgmt-state-store-${{ environment }}-rg
+        storageAccount: dtsmgmtstatestore${{ environment }}
+        container: ${{ environment }}     
+
+      jobs:
+        - job: TestAndPlan
```

name doesn't look right?
```suggestion
        - job: PlanAndApply
```

---

changed